### ISSUE TITLE
FIX - watch option

### ISSUE THREAD

**Issue by NossieUK** [OsaSoft/youtube-better-subscriptions#161]
Can you please make the mark watch option in the same place?  I feel like I'm playing whack-a-mole trying to click it and when I do I miss click and watch the video - thanks.

**Comment by al52025** [comment]
Im having this same problem and its killing me. It would be nice to be able to like right click the entire thumbnail to mark it as watched

**Comment by OsaSoft** [comment]
@aormond strange, you dont seem to have the channel avatar there. All mine show like this 
![obrazek](https://github.com/OsaSoft/youtube-better-subscriptions/assets/8895463/fa0116c6-7ec9-4d98-9e49-34942a42d6d2)

**Comment by al52025** [comment]
> @aormond strange, you dont seem to have the channel avatar there. All mine show like this ![obrazek](https://user-images.githubusercontent.com/8895463/272284577-fa0116c6-7ec9-4d98-9e49-34942a42d6d2.png)

Yup I have the same thing as aormond. The button is overlapping the author/view count and everytime I click it and it removes it it makes all the other videos jump around so I constantly have to watch where I'm clicking to hit the hide button.

**Comment by al52025** [comment]
I do have magic actions on and I think 1 other YouTube extensions. I know I had the problem before the 2nd extension I added but I can disable magic actions tonight and see if that's the issue and report back

**Comment by al52025** [comment]
> interesting. Do you have any other addons or is that a youtube setting that removes the channel avatar circle?

I turned off magic actions and custom tube and the watched button still jumps around. once the video gets removed and the other gets shifted around it will randomly resize them making the watched button go to a different location. Anything I could do to help figure out this bug. Its super annoying

**Comment by aormond** [comment]
Apologies. I forgot that I had Stylus installed with 'AdashimaaTube' to make the layout of the subscriptions page more like what it used to be. I also didn't realize that it was hiding the channel avatar (the 'Hide Channel Profile' option, for those that are curious).

**Comment by al52025** [comment]
> Apologies. I forgot that I had Stylus installed with 'AdashimaaTube' to make the layout of the subscriptions page more like what it used to be. I also didn't realize that it was hiding the channel avatar (the 'Hide Channel Profile' option, for those that are curious).

Is stylus a problem? I have that but I only have a simple filter on and that's to change visited urls to a different color so I can't imagine that would cause an issue. My other YouTube extensions were disabled and it was still jumping around

**Comment by al52025** [comment]
@OsaSoft Ive disabled 90% of my extensions and the watched button is still jumping around. I should also mention I'm using Brave so I decided to try it on chrome. Installed the extension and made sure it was the only extension turned on. Its still jumping around in Chrome too. Anything else we could try?

**Comment by Phantom1472** [comment]
Even with all other extensions disabled, this is still an issue:
![image](https://github.com/OsaSoft/youtube-better-subscriptions/assets/6776227/af5e789d-7e3d-4e79-a3f3-fce380e2a594)

It seems like the length of the title (if it's 1 row or 2 rows) is causing the button to be placed lower, I think this button was once displayed inside the thumbnail on the bottom left of the thumbnail, that was a better placement IMO, and it was always at the same place every time.

Please fix this.
Thanks.
